monkey/monkey_island/cc/resources/test/monkey_test_endpoint.py

```diff
@@ -6,7 +6,7 @@
 from monkey_island.cc.resources.auth.auth import jwt_required
 
 
-class MonkeyTest(flask_restful.Resource):
+class MonkeyTestEndpoint(flask_restful.Resource):
```

Comment:
Term `test` is still confusing. If possible, I'd rename the path to `monkey_island/cc/resources/blackbox/blackbox_monkey`. When you see the term blackbox it obvious what it means, meanwhile `test` doesn't tell a whole lot and also confuse you between unit test or different kind of test.